Wurst Client for MC 1.9.X/src/tk/wurst_client/mods/PanicMod.java

```diff
@@ -1,41 +1,52 @@
-/*
- * Copyright � 2014 - 2016 | Wurst-Imperium | All rights reserved.
- * 
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/.
- */
-package tk.wurst_client.mods;
-
-import tk.wurst_client.events.listeners.UpdateListener;
-import tk.wurst_client.mods.Mod.Category;
-import tk.wurst_client.mods.Mod.Info;
-
-@Info(category = Category.MISC,
-	description = "Instantly turns off all enabled mods.\n"
-		+ "Be careful with this!",
-	name = "Panic",
-	tags = "legit, disable",
-	help = "Mods/Panic")
-public class PanicMod extends Mod implements UpdateListener
-{
-	@Override
-	public void onEnable()
-	{
-		wurst.events.add(UpdateListener.class, this);
-	}
-	
-	@Override
-	public void onUpdate()
-	{
-		for(Mod mod : wurst.mods.getAllMods())
-			if(mod.getCategory() != Category.HIDDEN && mod.isEnabled())
-				mod.setEnabled(false);
-	}
-	
-	@Override
-	public void onDisable()
-	{
-		wurst.events.remove(UpdateListener.class, this);
-	}
-}
+/*
+ * Copyright � 2014 - 2016 | Wurst-Imperium | All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package tk.wurst_client.mods;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import tk.wurst_client.events.listeners.UpdateListener;
+import tk.wurst_client.mods.Mod.Category;
+import tk.wurst_client.mods.Mod.Info;
+
+@Info(category = Category.MISC,
+	description = "Instantly turns off all enabled mods.\n"
+		+ "Be careful with this!",
+	name = "Panic",
+	tags = "legit, disable",
+	help = "Mods/Panic")
+public class PanicMod extends Mod implements UpdateListener
+{
+	
+	private List<Mod> hiddenMods = new ArrayList<>();
+	
+	@Override
+	public void onEnable()
+	{
+		wurst.events.add(UpdateListener.class, this);
+	}
+	
+	@Override
+	public void onUpdate()
+	{
+		for(Mod mod : wurst.mods.getAllMods())
+			if(mod.getCategory() != Category.HIDDEN && mod.isEnabled())
+			{
+				mod.setEnabled(false);
```

Comment:
As you can see here, in the first loop it will disable all the mods, so the statement above will be always false.